MapboxCoreNavigation/MMEEventsManager.swift

```diff
@@ -135,6 +149,10 @@ struct EventDetails {
         modifiedEventDictionary["batteryPluggedIn"] = batteryPluggedIn
         modifiedEventDictionary["batteryLevel"] = batteryLevel
         modifiedEventDictionary["applicationState"] = applicationState.telemetryString
+        modifiedEventDictionary["userAbsoluteDistanceToDestination"] = userAbsoluteDistanceToDestination
+        modifiedEventDictionary["locationEngine"] = locationEngine is SimulatedLocationManager ? "SimulatedLocationManager" : locationEngine is NavigationLocationManager ? "NavigationLocationManager" : locationEngine.description
```

Comment:
`locationEngine.description` won't return what you expect here. Also, locationEngine might be a subclass of NavigationManager.

I tested in a playground and `String.init(describing: type(of: locationEngine))` seems to work.